crates/bitwarden-error-macro/src/basic/attribute.rs

```diff
@@ -51,7 +51,7 @@ fn basic_error_wasm(
             #[automatically_derived]
             impl From<#type_identifier> for JsValue {
                 fn from(error: #type_identifier) -> Self {
-                    let js_error = SdkJsError::new(error.to_string());
+                    let js_error = bitwarden_error::wasm::SdkJsError::new(error.to_string());
```

Comment:
I recommend using fully qualified names to avoid collisions with modules if possible
```suggestion
                    let js_error = ::bitwarden_error::wasm::SdkJsError::new(error.to_string());
```